ReservationDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using AirportTickets.Flight;
+
+namespace AirportTickets
+{
+    public class ReservationDTO
+    {
+        public string Name { get; private set; }
+        public FlightDTO Flight { get; private set; }
+
+        public ReservationDTO(string name, FlightDTO flight)
```

Comment:
assigning a flight object to reservation this way will be an issue for data integrity as mentioned before. What if you want to update the info of some flight? Then you to update the object in two places: `FlightRepository` and `BookingDTO`, since the flight is saved in two places/arrays.
We better instead save the flight id here in this constructor, and link by id instead of the object. If you want flight info through `ReservationDTO`, you just join with the original flight list in the repo.